src/Routes/Groups/GroupTable.js

```diff
@@ -39,15 +39,28 @@ const GroupTable = ({
   handleDeleteModal,
   hasModalSubmitted,
   setHasModalSubmitted,
+  locationProp,
+  navigateProp,
   fetchGroups,
 }) => {
   const hideCreateGroupsEnabled = useFeatureFlags(
     'edge-management.hide-create-group'
   );
 
   const history = useHistory();
-  const { pathname } = useLocation();
+  const { pathname } = locationProp
+    ? locationProp()
+    : useLocation
+    ? useLocation()
+    : null;
```

Comment:
if none of the options were true, this could lead to a `null` value.
Is there a better way to have a default value? like, initialize the prop?

Reply:
this is the case in all other compenent that we expose, I dont see other way except this inital

Reply:
do you have an suggestion to impement it in a different way?

Reply:
remove the null. thank you